graphql/api.js

```diff
@@ -159,3 +159,100 @@ export const allProfilesInfo = async () => {
     errorMessage: getErrorMessage(null, data)
   };
 };
+
+export const allApplicationsInfo = async () => {
+  const query = `query allApplicationsInfo {
+    allApplicationsInfo(_size: 8) {
+      data {
+        _id
+        company
+      }
+    }
+  }`;
+
+  const res = await fetch(process.env.NEXT_PUBLIC_FAUNADB_GRAPHQL_ENDPOINT, {
+    method: 'POST',
+    headers: {
+      Authorization: `Bearer ${process.env.NEXT_PUBLIC_FAUNADB_SECRET}`,
+      'Content-type': 'application/json',
+      Accept: 'application/json'
+    },
+    body: JSON.stringify({
+      query
+    })
+  });
+
+  const data = await res.json();
+
+  return {
+    data: getData(data),
+    errorMessage: getErrorMessage(null, data)
+  };
+};
+
+export const findApplicationByID = async (id) => {
+  const query = `query findApplicationByID($id: ID!){
+    findApplicationByID(id: $id){
+      company
+      role
+      profile {
+        name
+        account{
+          email
+        }
+      }
+    }
+  }`;
+
+  const res = await fetch(process.env.NEXT_PUBLIC_FAUNADB_GRAPHQL_ENDPOINT, {
+    method: 'POST',
+    headers: {
+      Authorization: `Bearer ${process.env.NEXT_PUBLIC_FAUNADB_SECRET}`,
+      'Content-type': 'application/json',
+      Accept: 'application/json',
+    },
+    body: JSON.stringify({
+      query,
+      variables: { id }
+    }),
+  });
+
+  const data = await res.json();
+
+  return {
+    data: getData(data),
+    errorMessage: getErrorMessage(null, data)
+  };
+};
+
+export const updateApplication = async (id, company, role, currentExcitement) => {
+  const query = `mutation updateApplication($id: ID!, $company: String!, $role: String!, $currentExcitement: Int!) {
+    updateApplication(id: $id, data:{
+      company: $company
+      role: $role
+      currentExcitment: $currentExcitement
+      }){
+        currentExcitment
+  		}
```

Comment:
tiny nit: indentation here seems off. (once i get around to adding the linter we wont need to worry about it anymore)